scss/components/_card.scss

```diff
@@ -81,6 +82,7 @@ $card-margin: $global-margin !default;
 ) {
   @if $global-flexbox {
     flex: 0 1 auto;
+    display: flex;
```

Comment:
Since this is a shrink container, it is the best place to add actions, images, or other pieces that will be typically distributed horizontally. Making this display: flex seems natural to apply flex helpers like `.align-justify` and has no adverse affects on elements inside (images will stretch if smaller than container height but `align-items` or `.align-middle` solves it).
![screen shot 2017-02-03 at 3 04 30 pm](https://cloud.githubusercontent.com/assets/5652085/22611907/1c565872-ea22-11e6-97c8-3665f9975cae.png)